rust_src/src/lisp.rs

```diff
@@ -939,6 +940,20 @@ impl LispObject {
             wrong_type!(Qstringp, self)
         }
     }
+
+    #[inline]
+    pub fn from_str(s: &str) -> LispObject {
+        LispObject::from(unsafe {
+            make_string(s.as_ptr() as *const c_char, s.len() as ptrdiff_t)
+        })
+    }
+
+    #[inline]
+    pub fn to_string(self) -> String {
+        let mut s = self.as_string().unwrap();
```

Comment:
Is that `unwrap` safe? Should the code error out?

Reply:
No. Yes. I'll replace the as_string w/as_string_or_error.

Thanks for the review.